src/Controller/GeneratePdf.php

```diff
@@ -59,7 +62,13 @@ public function handle(): string
         ]);
 
         $this->pdfCreator->create();
+        $pdf = $this->pdfCreator->generatePdf($template);
 
-        return $this->pdfCreator->generatePdf($template, 'brochure-community-event');
+        return new Response($pdf, 200,
+            [
+                'Content-Type' => 'application/pdf',
+                'Content-Disposition' => 'inline; filename="brochure-community-event-'.$this->date->format(\DateTime::ATOM).'.pdf"',
```

Comment:
No need here to add date directy into the constructor, you can simply add it into the handle and make (new \DateTime())->format(\DateTime::ATOM)
And use sprintf to insert the date on the filemane